TLSharp.Core/Auth/Step3_CompleteDHExchange.cs

```diff
@@ -206,4 +203,4 @@ public Step3_Response FromBytes(byte[] response)
             }
         }
     }
-}
+}
```

Comment:
why are you changing all the ending of files?

Reply:
i use CodeMaid Cleaner for cleaning code so this is just for increase the readability of the code

Reply:
i think there is a newline difference here 🤔🤔

Reply:
no new  code, just some usings removed (that dont used) and white spaces removed.